frame/aura/src/lib.rs

```diff
@@ -92,7 +92,12 @@ pub mod pallet {
 			if let Some(new_slot) = Self::current_slot_from_digests() {
 				let current_slot = CurrentSlot::<T>::get();
 
-				assert!(current_slot < new_slot, "Slot must increase");
+				if AllowMultipleBlocksPerSlot::<T>::get() {
```

Comment:
Should we increment the returned `Weight` as well? I.e. 3 reads instead of 2

Reply:
(will get back to this shortly. you are correct ofc)

Reply:
Outdated now that this no longer involves a storage read.